config/jobs/common/cla-assistant.yaml

```diff
@@ -3,6 +3,9 @@ periodics:
   interval: 10m
   cluster: gardener-prow-trusted
   decorate: true
+  reporter_config:
```

Comment:
Shall we do the same for `issue-pr-lifecycle.yaml`?

Reply:
Oh, yes, I'll include it here in a minute

Reply:
done 😃